### PR TITLE
Added documentation for Google Secret Manager secret engine 

### PR DESCRIPTION
**Pull request by kirangodishala**
Documentation for the recently added feature - Google Secret Manager implementation for secret engine.
https://github.com/spinnaker/kork/pull/960

### CODE REVIEW

**Comment by kirangodishala** [comment]
@dbyron-sf  @link108  @mattgogerly  - could you please review and merge this documentation PR.

**Review by sanopsmx — APPROVED** [review]
lgtm

File: content/en/docs/reference/halyard/secrets/secret-manager-secrets.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: "Secrets in Google Secret Manager"
+description: This document describes how to set up Spinnaker secrets in a Google Secret Manager.
+---
+
+This example uses secrets - `mysecret1`, `mysecret2` - to store GitHub credentials and  secret - `mykubeconfig` - to store kubeconfig file.
```
Comment (dbyron-sf) [inline content/en/docs/reference/halyard/secrets/secret-manager-secrets.md:6]:
```suggestion
This example uses secrets - `mysecret1`, `mysecret2` - to store GitHub credentials and secret - `mykubeconfig` - to store kubeconfig file.
```

**Review by dbyron-sf — COMMENTED** [review]
LGTM with one tiny nit.

**Comment by kirangodishala** [comment]
> LGTM with one tiny nit.

Corrected :-)

**Review by dbyron-sf — APPROVED** [review]

**Pull request merged**